### PR TITLE
Hardening/117 discovery convops

### PR DESCRIPTION
**Pull request by kzangeli**
ngsi9 convops for discovery (NEC convops) converted to use standard operations.

### CODE REVIEW

File: src/lib/serviceRoutines/getContextEntityTypeAttribute.cpp [hunk 1]
```diff
@@ -48,17 +56,29 @@ std::string getContextEntityTypeAttribute
...
 {
-  std::string                          entityType     = (compV[0] == "v1")? compV[3] : compV[2];
-  std::string                          attributeName  = (compV[0] == "v1")? compV[5] : compV[4];
-  DiscoverContextAvailabilityRequest*  requestP       = &parseDataP->dcar.res;
-  EntityId                             entityId(".*", entityType, "true");
+  std::string  entityType     = (compV[0] == "v1")? compV[3] : compV[2];
+  std::string  attributeName  = (compV[0] == "v1")? compV[5] : compV[4];
+  std::string  answer;
 
   LM_T(LmtConvenience,
```
Comment (fgalan) [inline src/lib/serviceRoutines/getContextEntityTypeAttribute.cpp]: In `getContextEntityAttributes()` there was a similar LM_T() that you have deleted... It is a bit uncoherent to have the LM_T() is come cases and not in the others, isn't it?

Comment (kzangeli) [inline src/lib/serviceRoutines/getContextEntityTypeAttribute.cpp]: Yeah, it's not very coherent, also not especially important ...

So, I'd remove all these LM_T, I can't say I use them anyway.

What do you think?

File: test/functionalTest/cases/117_convop_using_standard_ops/getEntityByIdAttributeByName.test [hunk 2]
```diff
@@ -0,0 +1,560 @@
+# Copyright 2015 Telefonica Investigacion y Desarrollo, S.A.U
```
Comment (fgalan) [inline test/functionalTest/cases/117_convop_using_standard_ops/getEntityByIdAttributeByName.test]: What about tests using pagination? Or maybe you consider pagination tested in the tests in a previous PR (https://github.com/telefonicaid/fiware-orion/pull/754)?

Comment (kzangeli) [inline test/functionalTest/cases/117_convop_using_standard_ops/getEntityByIdAttributeByName.test]: Of course, completely forgot about that!

File: src/lib/serviceRoutines/getContextEntityTypeAttribute.cpp (hunk 1 above)
Comment (fgalan) [inline src/lib/serviceRoutines/getContextEntityTypeAttribute.cpp]: Ok, let's do that. We can add them again if we need.

However, what about passed PRs? Are the LM_T removed them?

Comment (kzangeli) [inline src/lib/serviceRoutines/getContextEntityTypeAttribute.cpp]: I'll add a commit right now removing all of them

    grep LM_T src/lib/serviceRoutines/*.cpp | grep CONVENIENCE

Comment (kzangeli) [inline src/lib/serviceRoutines/getContextEntityTypeAttribute.cpp]: Fixed in 92c8543

File: test/functionalTest/cases/117_convop_using_standard_ops/getEntityByIdAttributeByName.test (hunk 2 above)
Comment (kzangeli) [inline test/functionalTest/cases/117_convop_using_standard_ops/getEntityByIdAttributeByName.test]: Fixed in a6adf86

**Comment by kzangeli** [comment]
make test  running ...

**Comment by fgalan** [comment]
LGTM

**Pull request merged**